### DISCUSSION TITLE
Vorta no longer list archives after client hostname change

### DISCUSSION THREAD

**Discussion by mxj4** [borgbase/vorta#1839] [FAQ]
My system got a new hostname recently, incremental backup still works, log says archives are found during pruning activities, but no archive is listed in the UI. Is there any hard-coded UI logic to filter archives by hostname?

**Comment by jetchirag** [comment]
Logic to create/list archives by default uses hostname. You can manually overwrite this in archives tab and replace hostname placeholder to the old one.

**Reply by mxj4** [reply]
I did change archive naming, but still no archive is visible in UI

**Reply by real-yfprojects** [reply]
Did you reload the archive list using the button on the bottom left?

**Reply by real-yfprojects** [reply]
> Logic to create/list archives by default uses hostname.

Afaik the logic to list archives doesn't use the hostname or the prefix. 

https://github.com/borgbase/vorta/blob/567a3546ae09878cf64f9f10c5c6f7bb7254720e/src/vorta/views/archive_tab.py#L279
